src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
-}
+  if (days < 3) {
+    return days * 40;
+  };
+
+  if (days >= 3 && days < 7) {
```

Comment:
What is this 3, 7? I can't understand what they mean. Use variables instead, [checklist](https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md) 1st point